mix.exs

```diff
@@ -1,29 +1,34 @@
-defmodule Tee.MixProject do
+defmodule Membrane.Element.Tee.MixProject do
   use Mix.Project
 
+  @version "0.1.0"
+  @github_url "https://github.com/membraneframework/membrane-element-tee"
+
   def project do
     [
-      app: :tee,
-      version: "0.1.0",
+      app: :membrane_element_tee,
+      version: @version,
       elixir: "~> 1.7",
       start_permanent: Mix.env() == :prod,
-      deps: deps()
+      source_url: @github_url,
+      deps: deps(),
+      docs: docs()
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
-  def application do
+  defp docs do
     [
-      extra_applications: [:logger]
+      main: "readme",
+      extras: ["README.md"],
+      source_ref: "v#{@version}"
     ]
   end
 
```

Comment:
You can add pacakge info here. 

https://github.com/membraneframework/membrane-element-rtp/blob/master/mix.exs#L48